fhir-server-test/src/test/java/com/ibm/fhir/server/test/SearchTest.java

```diff
@@ -228,7 +203,8 @@ private Person addIdentifiers(Person person, int numberOfIdentifiers) {
         return personBuilder.build();
     }
 
-    @Test(groups = { "server-search" }, dependsOnMethods = { "testCreatePatient" })
+    @Test(groups = { "server-search" }, dependsOnMethods = {
+    "testCreatePatient" })
```

Comment:
still don't like this formatting change

Reply:
ha, good catch. I have just fixed this and also removed the auto save action which has caused this. thanks!